src/elements/element.line.js

```diff
@@ -44,50 +42,51 @@
 
 			// Draw the background first (so the border is always on top)
 			helpers.each(this._children, function(point, index) {
+
 				var previous = helpers.previousItem(this._children, index);
 				var next = helpers.nextItem(this._children, index);
 
-				// First point only
-				if (index === 0) {
-					ctx.moveTo(point._view.x, point._view.y);
-					return;
+				// First point moves to it's starting position no matter what
+				if (!index) {
+					ctx.moveTo(point._view.x, vm.scaleZero);
 				}
 
-				// Start Skip and drag along scale baseline
-				if (point._view.skip && vm.skipNull && !this._loop) {
-					ctx.lineTo(previous._view.x, point._view.y);
-					ctx.moveTo(next._view.x, point._view.y);
+				// Skip this point, draw to scaleZero, move to next point, and draw to next point
+				if (point._view.skip && !this.loop) {
+					ctx.lineTo(previous._view.x, vm.scaleZero);
+					ctx.moveTo(next._view.x, vm.scaleZero);
+					return;
```

Comment:
Yes, I adopted a return fast pattern instead of a bunch of nested if else statements.